srv/components/misc_pkgs/rabbitmq/config/base.sls

```diff
@@ -37,8 +37,14 @@ Enable plugin rabbitmq_management:
     - name: rabbitmq_management
     - require:
       - Install RabbitMQ
-    - watch_in:
-      - Start RabbitMQ service
+
+Stop-Start RabbitMQ service:
+  module.run:
+    - service.dead:
+      - rabbitmq-server
+  module.run:
```

Comment:
No need for this. It could be simple:
```
Stop-Start RabbitMQ service:
  module.run:
    - service.dead:
      - rabbitmq-server
    - service.running:
      - rabbitmq-server
```

Reply:
addressed